### PR TITLE
tac: return UResult from uumain() function

### PR DESCRIPTION
**Pull request by jfinkels**
See #2464.

### CODE REVIEW

File: src/uu/tac/src/error.rs [hunk 1]
```diff
@@ -0,0 +1,56 @@
+//! Errors returned by tac during processing of a file.
```
Comment (sylvestre) [inline src/uu/tac/src/error.rs:1]:
```suggestion
//  * This file is part of the uutils coreutils package.
//  *
//  * For the full copyright and license information, please view the LICENSE
//  * file that was distributed with this source code.
//! Errors returned by tac during processing of a file.
```

**Comment by sylvestre** [comment]
some tests are broken but probably not because of you

**Comment by jfinkels** [comment]
I added the license header.

Looks like the build failure was due to a breaking change in a dependency? Not sure.

**Comment by sylvestre** [comment]
same, please remove cargo.lock

**Comment by jfinkels** [comment]
I have rebased and removed the `Cargo.lock` changes

**Pull request merged**